uwblueprint/sistering
# Prisma user and entity models

## Ticket link

Closes #14 



## Implementation description
* Generated Prisma schemas for `user` and `entity`
* Migrated `userService` from Sequelize to Prisma
* Migrated `entityService` from Sequelize to Prisma



## Steps to test
1. Go to http://localhost:5000/graphql and check that the `user` and `entity` CRUD services are still functional (any of login, users, userByEmail, createUser, updateUser, deleteUser, entities, entity, createEntity, updateEntity, deleteEntity)



## What should reviewers focus on?
* Correct usage of Prisma operations, correct Prisma schemas


## Checklist
